tests/lru_cache_test.py

```diff
@@ -78,77 +78,77 @@ def test_put_empty_key(self):
   def test_eviction(self):
     cache = LRUCache(self.name, max_size=2)
 
-    cache.put("cache-a", b"a")
-    cache.put("cache-b", b"b")
+    cache.put("a", b"a")
+    cache.put("b", b"b")
 
-    # `sleep()` is necessary to guarantee that `cache-b`"s timestamp is strictly greater than `cache-a`"s
+    # `sleep()` is necessary to guarantee that `b`"s timestamp is strictly greater than `a`"s
     time.sleep(1)
-    cache.get("cache-b")
+    cache.get("b")
 
-    # write `cache-c`, evict `cache-a`
-    cache.put("cache-c", b"c")
-    self.assertEqual(set(self.path.glob("cache-*")), {self.path / "cache-b", self.path / "cache-c"})
+    # write `c`, evict `a`
```

Comment:
Nit: the Google style guide normally requires comments to be proper sentences: https://google.github.io/styleguide/pyguide.html#386-punctuation-spelling-and-grammar.

JAX doesn't follow the style guide, so this comment is optional.

Reply:
Done